routers/api/v1/repo/file.go

```diff
@@ -136,8 +136,8 @@ func GetEditorconfig(ctx *context.APIContext) {
 	}
 
 	fileName := ctx.Params("filename")
-	def := ec.GetDefinitionForFilename(fileName)
-	if def == nil {
+	def, err := ec.GetDefinitionForFilename(fileName)
+	if err != nil {
```

Comment:
probably just ignore `err` as on error `def` should still be `nil`
```suggestion
	if def == nil {
```

Reply:
@lafriks I agree that it should be enough but err should not be ignored as they give context and would be reported in the not found.
Otherwise `ctx.NotFound(err)` is equivalent to `ctx.NotFound(nil)` in this context.
Looking at [GetDefinitionForFilename](https://github.com/editorconfig/editorconfig-core-go/blob/b89dbeb63d921bde3419e701ff1051850b3a0b3a/editorconfig.go#L292) matching on `err != nil` seems better at first glance or at least not ignoring it would still fill the err value to be exposed via the next line.